.github/workflows/ort.yml

```diff
@@ -3,19 +3,91 @@ name: ORT
 on:
   workflow_dispatch:
 
-env:
-  ORT_IMAGE: ghcr.io/oss-review-toolkit/ort-minimal
-
 jobs:
   ort:
     name: Run ORT
     runs-on: ubuntu-22.04
     steps:
     - name: Checkout Repository
       uses: actions/checkout@v4
+      with:
+        path: ort-server
+
+    - name: Setup pnpm
+      uses: pnpm/action-setup@v4
+      with:
+        version: 9
+
+    - name: Setup Node
+      uses: actions/setup-node@v4
+      with:
+        node-version: 20
```

Comment:
Should we also have `cache: pnpm` here?

Reply:
I wasn't sure if this would have an effect, but as ORT calls `pnpm install` it should actualy help. I have now also added the setup-gradle action for the same reason, but we will only see if this has an effect after the merge, because AFAIK both actions only write to the cache from the main branch.